docs/book/src/reference/versions.md

```diff
@@ -43,47 +43,44 @@ These diagrams show the relationships between components in a Cluster API releas
+| Kubernetes v1.22 |                                  | ✓                             | ✓                                | ✓                             |
 
 The Core Provider also talks to API server of every Workload Cluster. Therefore, the Workload Cluster's Kubernetes version must also be compatible.
 
 #### Kubeadm Bootstrap Provider (`kubeadm-bootstrap-controller`)
 
-|                                    | Cluster API v1alpha2 (v0.2) | Cluster API v1alpha3 (v0.3) |
-| ---------------------------------- | --------------------------- | --------------------------- |
-| Kubernetes v1.13                   |                             |                             |
-| Kubernetes v1.14 + kubeadm/v1beta1 | ✓                           |                             |
-| Kubernetes v1.15 + kubeadm/v1beta2 | ✓                           |                             |
-| Kubernetes v1.16 + kubeadm/v1beta2 | ✓                           | ✓                           |
-| Kubernetes v1.17 + kubeadm/v1beta2 |                             | ✓                           |
-| Kubernetes v1.18 + kubeadm/v1beta2 |                             | ✓                           |
-| Kubernetes v1.19 + kubeadm/v1beta2 |                             | ✓                           |
-| Kubernetes v1.20 + kubeadm/v1beta2 |                             | ✓                           |
+|                                    |  CAPI v1alpha3 (v0.3) Management | CAPI v1alpha3 (v0.3) Workload | CAPI v1alpha4 (v0.4) Management | CAPI v1alpha4 (v0.4) Workload |
+| ---------------------------------- | -------------------------------- | ----------------------------- | ------------------------------- | ----------------------------- |
+| Kubernetes v1.16 + kubeadm/v1beta2 | ✓                                | ✓                             |                                 | ✓                             |
+| Kubernetes v1.17 + kubeadm/v1beta2 | ✓                                | ✓                             |                                 | ✓                             |
+| Kubernetes v1.18 + kubeadm/v1beta2 | ✓                                | ✓                             |                                 | ✓                             |
+| Kubernetes v1.19 + kubeadm/v1beta2 | ✓                                | ✓                             | ✓                               | ✓                             |
+| Kubernetes v1.20 + kubeadm/v1beta2 | ✓                                | ✓                             | ✓                               | ✓                             |
+| Kubernetes v1.21 + kubeadm/v1beta2 | ✓                                | ✓                             | ✓                               | ✓                             |
+| Kubernetes v1.22 + kubeadm/v1beta2 |                                  | ✓                             | ✓                               | ✓                             |
```

Comment:
This should be v1beta3 with v0.4 but still v1beta2 for v0.3

(@fabriziopandini that's correct, right?)

Reply:
yes, lets add a note